docs/execution_providers/TensorRT-ExecutionProvider.md

```diff
@@ -7,20 +7,35 @@ With the TensorRT execution provider, the ONNX Runtime delivers better inferenci
 ## Build
 For build instructions, please see the [BUILD page](../../BUILD.md#tensorrt). 
 
-The TensorRT execution provider for ONNX Runtime is built and tested with TensorRT 6.0.1.5 but validated with the feature set equivalent to TensorRT 5. Some TensorRT 6 new features such as dynamic shape is not available as this time.
+The TensorRT execution provider for ONNX Runtime is built and tested with TensorRT 6.0.1.5.
 
 ## Using the TensorRT execution provider
 ### C/C++
-The TensortRT execution provider needs to be registered with ONNX Runtime to enable in the inference session. 
+The TensorRT execution provider needs to be registered with ONNX Runtime to enable in the inference session. 
 ```
 InferenceSession session_object{so};
 session_object.RegisterExecutionProvider(std::make_unique<::onnxruntime::TensorrtExecutionProvider>());
 status = session_object.Load(model_file_name);
 ```
 The C API details are [here](../C_API.md#c-api).
 
+#### Sample
+To run Faster R-CNN model on TensorRT execution provider,
```

Comment:
it's better to do a notebook tutorial to be consistent with other EPs. 
let's do it as a separate future PR.

Reply:
Will do